README.md

```diff
@@ -13,61 +13,8 @@ The website is available under the following domains, it is preferred to use the
 
-# Building frc-docs
-frc-docs uses the Sphinx documentation generator, and documentation is written in [reStructuredText](http://docutils.sourceforge.net/rst.html).
+# Contributing to frc-docs
 
-Ensure the repository is cloned with ``git clone --depth 1 https://github.com/wpilibsuite/frc-docs.git``.
+We're happy to have you contributing to frc-docs, whether it's; writing articles, fixing code, new extensions, or adding translations!
 
-## Requirements
-- **Windows**
-  - [Python 3.6 or greater](https://www.python.org/downloads/)
-  - [MiKTeX](https://miktex.org/download)
-  - [Perl](http://strawberryperl.com/)
-
-- **Linux (Ubuntu)**
-  - ``sudo apt-get install -y python3 python3-pip``
-  - ``sudo apt-get install -y texlive-latex-recommended texlive-fonts-recommended texlive-latex-extra latexmk texlive-lang-greek texlive-luatex texlive-xetex texlive-fonts-extra dvipng librsvg2-bin``
-
-Ensure the Python requirements are installed via running ``python3 -m pip install -r source/requirements.txt``
-
-**Note:** Windows users who are building PDF or EPUB versions of the docs *must* run ``mpm --verbose --require=@miktex-packages.txt`` to properly install all of the required MiKTeX packages.
-
-## Building
-
-### Lint Check
-
-``.\make lint`` or ``make lint``
-
-**Note**: Due to a bug with the linter on Windows machines, this command is currently only usable on Linux systems.
-
-### Image Size Check
-
-``.\make sizecheck`` or ``make sizecheck``
-
-This checks all images are under 500KB
-
-### Redirects Check
-
-Files that have been moved or renamed must have their new location (or replaced with 404) in the ``redirects.txt`` file in ``source``. To verify that there are no invalid redirects. Run:
-
-``.\make rediraffecheckdiff`` or ``make rediraffecheckdiff``
-
-Additionally, an HTML build may need to be ran to verify that all redirects build properly.
-
-### Building HTML
-
-``.\make html`` or ``make html``
-
-### Building PDF
-
-``.\make latexpdf`` or ``make latexpdf``
-
-**Note**: Run the ``scripts/install-rsvg-convert.ps1`` script in powershell to ensure ``rsvg-convert`` is installed.
-
-### Building EPUB
-
-``.\make epub`` or ``make epub``
-
-## Additional Note
-
-A web version of these instructions is available at the [frc-docs website](https://docs.wpilib.org/en/latest/docs/contributing/frc-docs/build-instructions.html).
+All the information needed to contribute to frc-docs is located in the ![contributing](https://docs.wpilib.org/en/latest/docs/contributing/frc-docs/index.html) section on the documentation website. Additionally, please ensure that all contributions follow the WPILib ![Code of Conduct](https://github.com/wpilibsuite/allwpilib/blob/master/CODE_OF_CONDUCT.md).
```

Comment:
I think frc-docs website rather then documentation website is better to be consistent with everywhere else.